conduct.md

```diff
@@ -132,7 +132,9 @@ TODO: Write an introductory paragraph.
 
 ### Infractions
 
-TODO: Write an introductory paragraph.
+Infractions are violations of the honor code and campus policies. Depending upon the infraction one commits and past infractions, different penalties are applied to the student.
```

Comment:
In the context of this conduct guide, infractions are violations of specifically this conduct guide. This conduct guide might say that students must uphold institutional policies, such as the Honor Code, but infractions here should be defined as violations of this conduct guide. The reason this distinction is important is because we cannot define our own penalties for institutional policies, such as the Honor Code, which already have a disciplinary system.